tests/test_helpers.py

```
@@ -32,14 +31,14 @@ def test_create_json_folder_param(tmp_path: Path | Any, precreate: bool) -> None
 
 @pytest.mark.parametrize(
-    "existing,expected",
+    ("existing", "expected"),
     [
         (True, {"a": 1, "b": "x"}),
         (False, {}),
     ],
 )
 def test_get_dict_from_json_file_param(
-    tmp_path: Path | Any, existing: bool, expected: dict[str, Any]
+    tmp_path: Path, existing: bool, expected: dict[str, int] | dict[str, str]
```

Ruff (0.15.12)

[warning] Boolean-typed positional argument in function definition

(FBT001)